client/my-sites/plans/navigation.js

```diff
@@ -163,4 +169,14 @@ class PlansNavigation extends React.Component {
 	}
 }
 
-export default localize( PlansNavigation );
+export default connect( state => {
+	const site = getSelectedSite( state );
+	const siteId = site ? site.ID : null;
```

Comment:
Why not use the `getSelectedSiteId` selector instead? It directly queries the state.

Reply:
It was too tempting to just pluck this value, but you're right, I'll swap it.

Reply:
49059cfb09